source/install/install-mmte-helm-gitlab-helm.rst

```diff
@@ -0,0 +1,238 @@
+        - name: POSTGRES_HOST_GITLAB
+          value:<GITLAB.POSTGRES.HOST>
+        - name: POSTGRES_PORT_GITLAB
+          value: "<GITLAB.POSTGRES.PORT>"
+        - name: POSTGRES_DB_NAME_MATTERMOST
+          value: <MATTERMOST.DATABASE.NAME>
+      command:
+        - sh
+        - "-c"
+        - |
+          if PGPASSWORD=$POSTGRES_PASSWORD_GITLAB psql -h $POSTGRES_HOST_GITLAB -p $POSTGRES_PORT_GITLAB -U $POSTGRES_USER_GITLAB -lqt | cut -d \| -f 1 | grep -qw $POSTGRES_DB_NAME_MATTERMOST; then
+          echo "database already exist, exiting initContainer"
+          exit 0
+          else
+          echo "Database does not exist. creating...."
+          PGPASSWORD=$POSTGRES_PASSWORD_GITLAB createdb -h $POSTGRES_HOST_GITLAB -p $POSTGRES_PORT_GITLAB -U $POSTGRES_USER_GITLAB $POSTGRES_DB_NAME_MATTERMOST
+          echo "Done"
+          fi
+    - name: create-minio-bucket
+      image: "minio/mc:RELEASE.2018-07-13T00-53-22Z"
+      env:
+        - name: MINIO_ENDPOINT
+          value: <GITLAB.MINIO.HOST>
+        - name: MINIO_PORT
+          value: "<GITLAB.MINIO.PORT>"
+        - name: MINIO_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: accesskey
+        - name: MINIO_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: secretkey
+        - name: MATTERMOST_BUCKET_NAME
+          value: <MATTERMOST.MINIO.BUCKET.NAME>
+      command:
+        - sh
+        - "-c"
+        - |
+          echo "Connecting to Minio server: http://$MINIO_ENDPOINT:$MINIO_PORT"
+          mc config host add myminio http://$MINIO_ENDPOINT:$MINIO_PORT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+          /usr/bin/mc ls myminio
+          echo $?
+          /usr/bin/mc ls myminio/$MATTERMOST_BUCKET_NAME > /dev/null 2>&1
+          if [ $? -eq 1 ] ; then
+            echo "Creating bucket '$MATTERMOST_BUCKET_NAME'"
+            /usr/bin/mc mb myminio/$MATTERMOST_BUCKET_NAME
+          else
+            echo "Bucket '$MATTERMOST_BUCKET_NAME' already exists."
+            exit 0
+          fi
+
+Values that you need to replace in the above ``values.yaml`` file:
+
+- **<YOUR.MATTERMOST.DOMAIN>**: URL that users will use to access Mattermost, matching the `Site URL field <https://docs.mattermost.com/administration/config-settings.html#site-url>`_, e.g. ``mattermost.gitlab.example.com``.
+- **<NAME.OF.YOUR.TLS.SECRET>**: A name to store the TLS certificate for you domains, e.g. ``mattermost-tls``.
+- **<INGRESS.CLASS>**: The ingress class. In a basic GitLab deployment, this is ``gitlab-nginx``.
+- **<INGRESS.PROVIDER>**: The ingress provider. In a basic GitLab deployment, this is ``nginx``.
+- **<CERTMANAGER.ISSUER>**: The cert manager issuer. In a basic GitLab deployment, this is ``gitlab-issuer``.
+- **<GITLAB.APP.SECRET>**: The Application secret, which you created in step `Create the OAUTH with GitLab`_.
```

Comment:
Do these `Create the OAUTH with GitLab` links work?

Reply:
@amyblais thoughts on this one?

Reply:
it worked when I test locally